rdr_service/gcloud_functions/retention_eligible_import_function/main.py

```diff
@@ -78,12 +78,17 @@ def retention_eligible_import_function(_event, _context):
     """
     GCloud Function Entry Point (Storage Pub/Sub Event).
     https://cloud.google.com/functions/docs/concepts/events-triggers#functions_parameters-python
-    :param event: (dict):  The dictionary with data specific to this type of event.
+    :param _event: (dict):  The dictionary with data specific to this type of event.
                        The `data` field contains a description of the event in
                        the Cloud Storage `object` format described here:
                        https://cloud.google.com/storage/docs/json_api/v1/objects#resource
-    :param context: (google.cloud.functions.Context): Metadata of triggering event.
+    :param _context: (google.cloud.functions.Context): Metadata of triggering event.
     """
     with GCPCloudFunctionContext(function_name, None) as gcp_env:
-        func = RetentionEligibleImportFunction(gcp_env, _event, _context)
-        func.run()
+        # checking to see if the file being dropped in the bucket is the retention file before executing the function
+        file_name = _event['name']
+        if 'retention' in file_name:
+            func = RetentionEligibleImportFunction(gcp_env, _event, _context)
+            func.run()
+        else:
+            _logger.info(f"skipping file: {file_name}, as it is not the retention file.")
```

Comment:
We generally don't want to short-circuit the class here in the Cloud function since this function can be used for a variety of file types. The `created` method is where this logic should live since we can build on that later.